build_r.R

```diff
@@ -5,6 +5,9 @@
 # Sys.setenv("CXX" = "/usr/local/bin/g++-8")
```

Comment:
this is not strictly related to this PR and is being added in #2957 . It'll disappear from the diff when that gets merged

Reply:
now that #2957 has been merged, I'll rebase to `master` before the next commit I push here